Readme.md

```diff
@@ -1,6 +1,6 @@
 # Equipment watchdog
```

Comment:
- [ ] Style equipment
- [ ] Size equipment in edit member
- [ ] Make reactive to save. How?